.github/workflows/skyportal-fink-client.yml

```diff
@@ -0,0 +1,161 @@
+name: skyportal-install
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
```

Comment:
This timeout seems quite big. What is the reason behind?

Reply:
Nothing specific. We took it from skyportal's GA. Because of all of its dependencies, SkyPortal can take a long time to initialized, but 90 minutes is too much. Maybe we could try reducing it progressively ? We can have a look at the highest time it ever took to run the GA here and set it at the timeout @JulienPeloton ? (10 minutes should be enough)